lib/ansible/modules/system/systemd.py

```diff
@@ -494,8 +495,12 @@ def main():
                         if rc != 0:
                             module.fail_json(msg="Unable to %s service %s: %s" % (action, unit, err))
             else:
-                # this should not happen?
-                module.fail_json(msg="Service is in unknown state", status=result['status'])
+                # check for chroot
+                if is_chroot():
+                    module.warn("Target is a chroot. This can lead to false positives or prevent the init system tools from working." % unit)
```

Comment:
i'm fine with adding the warning, but why change the failure condition?

Reply:
derp - i misunderstood your request for changes